api/src/main/java/com/intern/api/config/MongoConf.java

```diff
@@ -0,0 +1,26 @@
+package com.intern.api.config;
+
+import com.intern.api.config.properties.MongoProperties;
+import com.mongodb.MongoClient;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.bson.Document;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@Slf4j
+@RequiredArgsConstructor
+public class MongoConf {
+
+  final MongoProperties mongoProp;
```

Comment:
Java notations don't like cropped words and abbreviations. Also fix it for class name.